lib/dynflow/action.rb

```diff
@@ -89,6 +89,13 @@ def to_s_humanized
       end
     end
 
+    DelayedEvent = Algebrick.type do
```

Comment:
New `DelayedEvent` class, for not abusing `Director::Event` and to validate the time properly.